### ISSUE TITLE
[MU4 Issue] [Accessibility] Screen reader should announce pitch changes when moving note up and down

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#10453]
https://user-images.githubusercontent.com/90187801/152572157-af812d48-0e24-4a3a-9fa5-61f120128882.mp4

**Platform information**
 macOS, Windows, Linux

**Comment by DmitryArefiev** [comment]
Fixed in #12308
